README.md

```diff
@@ -1,73 +1,34 @@
-In which case the code user just needs a pointer there - e.g. [at_client on pub.dev](https://pub.dev/packages/at_client)
-
-### Contributor
-
-This is the person who we want working with us here.
-[CONTRIBUTING.md](CONTRIBUTING.md) is going to have the detailed guidance on how to setup their tools,
-tests and how to make a pull request.
-
-## Why, What, How?
-
-### Why?
-
-What is the purpose of this project?
-
-### What?
-
-What is needed to get the project and its dependencies installed?
-
-### How?
-
-How does this work? How is this used to fulfil its intended purpose?
-
-## Checklist
-
-### Writing
-
-Does the writing flow, with proper grammar and correct spelling?
-
-### Links
-
-Are the links to external resources correct?
-Are the links to other parts of the project correct
-(beware stuff carried over from previous repos where the
-project might have lived during earlier development)?
-
-### Description
-
-Has the Description field been filled out?
-
-### Acknowledgement/Attribution
-
-Have we correctly acknowledged the work of others (and their Trademarks etc.)
-where appropriate (per the conditions of their LICENSE?
-
-### LICENSE
-
-Which LICENSE are we using?  
-Is the LICENSE(.md) file present?  
-Does it have the correct dates, legal entities etc.?
-
-## Maintainers
-
-Who created this?  
-
-Do they have complete GitHub profiles?  
-
-How can they be contacted?  
-
-Who is going to respond to pull requests?  
+# Rust SDK - (⚠️Alpha version⚠️)
+This repo contains libraries, tools, samples and examples for developers who wish to work with the atPlatform from Rust code.
+
+It currently has limited functionality with minimal tests.
+
+## Run examples
+### Send data
+Send data to an AtSign
+```sh 
+RUST_LOG=info cargo run --example send_data_example ~/.atsign/keys/@aliens12_key.atKeys hello_there aliens12 virgogigantic64
+```
+
+### Fetch data
+Fetch data from an AtSign
+```sh 
+RUST_LOG=info cargo run --example fetch_data_example ~/.atsign/keys/@virgogigantic64_key.atKeys virgogigantic64 aliens12
+```
+
+## Structure
+- `at_client` - What consumers of the library will mostly interact with
+- `at_secrets` - Struct for constructing secrets from a file
+- `at_chops` (Cryptographic and Hashing Operations (CHOPS))
+    - `utils.rs` - Contains the generic, low level crypto operations
+    - `at_chops.rs` - Contains the specific combination of crypto operations that the client and verbs can use
+- `verbs` - Contains a trait that all verbs have to implement. Verbs execute the at protocol verbs by taking in arguments from the client.
+
+## Logging
+This library uses the `log` crate. This means implementors of this library can use something like `env_logger` and get info from the library.
+
+## Contributions welcome!
+
+All of our software is open with intent. We welcome contributions - we want pull requests, and we want to hear about issues. See also [CONTRIBUTING.md](CONTRIBUTING.md).
```

Comment:
What should be added/changed?

Reply:
The main thing is to replace `REPO` with `at_rust`, but also take a look at the rest and see if anything needs changing.

Reply:
I have updated CONTRIBUTING.md and README.md to include the pre-requisites. This just states to have Rust installed with a link to the docs on how to do so. The docs mention you might need to install `build-essential` on Linux. I also believe it will install the other dependencies you mentioned (openssl, libssl-dev, pkg-config).

Reply:
I had to install openssl, libssl-dev, pkg-config manually. Rustup and Cargo didn't make it happen :/

Reply:
Oh :/ I shall udpate to include this info then.